cockpit/cockpit-greetings-plugin/rollup.config.js

```diff
@@ -0,0 +1,21 @@
+import babel from "@rollup/plugin-babel";
+import resolve from "@rollup/plugin-node-resolve";
+import commonjs from "@rollup/plugin-commonjs";
+import replace from "@rollup/plugin-replace";
+
+export default {
+  input: "src/plugin.js",
+  output: {
+    file: "dist/plugin.js"
+  },
+  plugins: [
+    resolve(),
+    babel({"presets": ["@babel/preset-react"]}),
```

Comment:
```suggestion
    babel({
      presets: ["@babel/preset-react"],
      compact: false,
      babelHelpers: "bundled",
    }),
```

I think the user experience is nicer if no warning is shown when building the plugin. 🙂